content/_releases/22/18.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Product Update - v22.18
+description: Details of the product update v22.18 from 5th of May 2022.
+layout: rel
+version: "22.18"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-05-05
+redirect_from:
+  - /releases/22.18.html
+---
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements done in this round.
+
+### Configure login attempts
+
+Now you can configure the number of failed login attempts your users can have before
+locking them out of the system. We added a new variable `MAX_LOGIN_ATTEMPTS` to the
+HELM3 secrets chart. Set this variable any reasonable number for your system, otherwise
+it defaults to 5. Here is how we calculate this number:
+
+*   In case you don't have 2FA enabled, this is the number of failed login attempts.
+*   In case you have 2FA enabled, this is combined number of failed 2FA code and login attempts. This means if your user has 2FA enabled and succeeded with login but failed with 2FA more than the value of `MAX_LOGIN_ATTEMPTS` then your user will be locked out of the system. To unlock such users the tenant administration must first [disable 2FA for this user]({{site.data.tenant.apiDocsUri}}/v2/#disable-two-factor-authentication-(totp)-for-a-user) and instruct users to navigate `/forgot` address of the tenant to reset their password. This will reset the counter and user can login again.
+
+
+## OEM Related
+
+News in this section are for our customers who use
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### Removing docker registry builds on component deletion
+
+The platform builds a component docker image every time you push a new version of
+your component. When you remove the component or a particular version of it from
+the platform, an associated docker build would remain until you would remove it at
+later stage directly from the docker registry.
+
+With platform {{page.version}} we improved the situation in the following way:
+*   When you remove a version of your component, the platform removes the associated docker build.
+*   When you remove your component entirely, the platform removes all builds of your component.
+*   We created a special tool (`remove-deleted-repos-from-docker`) to help cleanup the remnant builds. Get in touch with us to get a copy of it.
```

Comment:
Get in touch -> Contact us
Add a link to 'Contact us'

Reply:
nop

Reply:
although ... ok